activity/urls.py

```diff
@@ -163,18 +163,17 @@
     path('bookmark_list', BookmarkList.as_view(),
          name='bookmark_list'),
     path('bookmark_add', BookmarkCreate.as_view(), name='bookmark_add'),
-    re_path(r'^bookmark_update/(?P<pk>\w+)/$',
-            BookmarkUpdate.as_view(), name='bookmark_update'),
-    re_path(r'^bookmark_delete/(?P<pk>\w+)/$',
-            BookmarkDelete.as_view(), name='bookmark_delete'),
+    path('bookmark_update/<pk>/',
+         BookmarkUpdate.as_view(), name='bookmark_update'),
+    path('bookmark_delete/<pk>/',
+         BookmarkDelete.as_view(), name='bookmark_delete'),
 
     # Auth backend URL's
     path('', include(('django.contrib.auth.urls',
                       "django.contrib.auth"), namespace='auth')),
-    re_path(r'^activate/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]'
-            r'{1,13}-[0-9A-Za-z]{1,20})/$',
-            views.activate_acccount, name='activate'),
+    path('activate/<uidb64>/<token>/',
```

Comment:
Also here `/<slug:uidb64>/<slug:token>/`

Reply:
@odenypeter  I have update the urls as requested